experimental/packages/opentelemetry-exporter-prometheus/test/PrometheusExporter.test.ts

```diff
@@ -331,11 +346,14 @@ describe('PrometheusExporter', () => {
       });
     });
 
-    it('should export a comment if no metrics are registered', async () => {
+    it('should export resource even if no metrics are registered', async () => {
       const body = await request('http://localhost:9464/metrics');
       const lines = body.split('\n');
 
-      assert.deepStrictEqual(lines, ['# no registered metrics']);
+      assert.deepStrictEqual(lines, [
```

Comment:
Looks like this target info metric default output invalidates the condition for no metrics here: https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/opentelemetry-exporter-prometheus/src/PrometheusExporter.ts#L199

Should we move the logic to the serializer?

Reply:
Yep, that's a good idea. Done in [b22e4cd](https://github.com/open-telemetry/opentelemetry-js/pull/3300/commits/b22e4cd075368ecda63b0130e7577545db2d8a90) :slightly_smiling_face: